lib/metadata.mjs

```diff
@@ -0,0 +1,51 @@
+/*
+  Copyright 2024 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import {execa} from 'execa';
+import fs from 'fs-extra';
+import {join} from 'path';
+import {packageShortNameRegex, withLogs} from './util.mjs'
+
+// Creates docs.metadata, based on package.json and .repo-metadata.json.
+export default async function createMetadata(opts = {}) {
+  const cwd = opts.cwd || process.cwd();
+  const destination = opts.destination || join(process.cwd(), '_devsite'); 
+  const packageInfo = await fs.readJson(join(cwd, 'package.json'));
+  const packageShortName = packageInfo.name.replace(packageShortNameRegex, '');
```

Comment:
Should this raise an error if the regex doesn't match so we don't have this problem pop up again? There might be other packages that don't start with `@google-`.

https://github.com/googleapis/nodejs-cloud-rad/blob/f8cafa3915794092c910207f42854ab6623c2a30/lib/util.mjs#L21

Reply:
Yes -- good idea! I'll throw an error for packages that don't have an `@google-` prefix.

Reply:
Ah, we do have some namespaces that are @google* (without the hyphen), as well as @googleapis - so maybe just an error for those that don't start with @google.

Reply:
Thanks for pointing that out -- updated the regex.